zad4/js/scripts.js

```diff
@@ -0,0 +1,12 @@
+function drawTree(a) {
+	for ( var i = 1 ; i <= a; i++ ) {
+		var star = "";
+		for (var j = 1; j <= i; j++) {
+			var	star = star + '*';
```

Comment:
nie musisz drugi raz deklarowac zmiennej `star` za pomoca slowa kluczowego `var`, bo masz ja juz wczesniej zadeklarowana

Reply:
to usunęłam zmienne. Ale dlaczego pojawia mi się w konsoli 'undefined'?

Reply:
zwroc uwage, ze komentarz wstawiam pod konkretna linia w kodzie, ktorej dotyczy. Tutaj nic sie nie zmienilo, jest drugi raz `var star`

Reply:
zostalo tylko usuniecie tego `var` w linii 5 poniewaz sluzy do pierwszego zadeklarowania zmiennej, a masz to juz zrobione w linii 3. W linii 5 chcesz tylko nadpisac zmienna nowymi danymi